### ISSUE TITLE
Add road type field to most datasets, to aid rendering

### ISSUE THREAD

**Issue by mvl22** [nptscot/nptscot.github.io#205]
Most of the line-based layers have the problem that residential/unclassified streets show prominently because they naturally have good cycleability, to the detriment of the main roads.

At present, we have nothing in these datasets to switch on the status of the road, i.e. main vs non-main roads.

So it would be useful to ensure that these datasets all have a field added giving the road status. We can then make various styling changes, in particular making the residential/unclassified streets have a narrower line width. That will massively improve the display in various cases. The road status would not show in popups, etc., rather it would merely be used as a styling hint.

Basically if you can dump into the data a new field which specifies if the Way is a main road*  or not, I can then theoretically style the non-main roads thinner than the main roads.

* (`motorway`/`trunk`/`primary`/`secondary`/`tertiary`, and `motorway_link`/`trunk_link`/`primary_link`/`secondary_link`/`tertiary_link`)

This was originally raised in https://github.com/nptscot/nptscot.github.io/issues/193#issuecomment-2140728648:

> Cycling by design compliance
> We need some way to separate out the data, maybe involving filtering out the majority of residential roads?

**Comment by Robinlovelace** [comment]
Yes, this is an important one..

**Comment by Robinlovelace** [comment]
Let's add `highway` to the CbD layer first, right? Would be a bigger job for the `rnet` layers.

**Comment by Robinlovelace** [comment]
The data is in the latest version of the pmtiles. See commits in #208

**Comment by mvl22** [comment]
> Let's add highway to the CbD layer first, right?

Thanks. I'll need to add support next for a field to have a styling based on another field, so that can be specified at:

https://github.com/nptscot/nptscot.github.io/blob/dev/src/datasets.js#L199
